schema/test/unit/tables/project_revision_test.sql

```diff
@@ -134,6 +134,24 @@ select is(
   'project_revision_001_add_revision_type adds the General Revision default revision_type to revisions that do not have a type');
 
 
+-- deploy project_revision_002_set_revision_statuses
+alter table cif.project_revision disable trigger _100_committed_changes_are_immutable, disable trigger _100_timestamps;
+
+alter table cif.project_revision alter column revision_status set default 'Draft';
+
+update cif.project_revision
+set revision_status =
+(case
+      when
+        (change_status='pending') or
+        (change_status='staged')
+        then 'Draft'
+    else 'Applied'
+    end);
+
+alter table cif.project_revision enable trigger _100_committed_changes_are_immutable, enable trigger _100_timestamps;
+
```

Comment:
This should already have been deployed by the migrations - also, there aren't any tests after it, what is this testing? Or are we just making sure this runs?

Reply:
I moved the `project_revision`-related tests to this file. I've copy-pasted (ish) the code from `project-revision` because running an `sql` file from inside a test file causes problems (I forget exactly what, something to do with transactions I think, Dylan and I came across it when writing the test for `project_revision_001_add_revision_type`